### ISSUE TITLE
[feature request] discord bot to message a channel on a serer instead of PM

### ISSUE THREAD

**Issue by JunkyardCat** [steve1316/granblue-automation-pyautogui#111]
is it possible for the bot to message a channel instead of DM and he will like mention everyone/specific users if it encounters error

**Comment by steve1316** [comment]
Yes, it is possible to have the bot send a message to a channel instead. But that is outside the scope of what I wanted this program to do but I will accept pull requests for this.
